### PR TITLE
Fix commented code

### PR DESCRIPTION
**Pull request by whotwagner**
# Make sure these boxes are signed before submitting your Pull Request -- thank you.

# Must haves
- [ ] I have read and followed the contributing guide lines at https://github.com/ait-aecid/logdata-anomaly-miner/wiki/Git-development-workflow
- [ ] Issues exist for this PR
- [ ] I added related issues using the "Fixes #<issue-id>"-notations
- [ ] This Pull-Requests merges into the "development"-branch

Fixes Deepsource Issues

# Submission specific

- [ ] This PR introduces breaking changes
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

# Describe changes:
- Removed commented code

### CODE REVIEW

File: aecid-testsuite/unit/analysis/NewMatchPathDetectorTest.py [hunk 1]
```diff
@@ -207,48 +181,6 @@ def test10_allowlist_event_with_known_and_unknown_paths(self):
...
-    #
-    # '''
-    # The data type must be checked before calculating the remaining time Requires type check (not yet implemented).
-    # '''
-    # def test14_fuzzing_trigger_time(self):
-    #   self.new_match_path_detector = NewMatchPathDetector(self.aminer_config,
-    #     [self.stream_printer_event_handler], 'Default', True)
-    #   self.new_match_path_detector.nextPersistTime = 400
-    #   self.assertRaises(ArgumentTypeError, self.new_match_path_detector.do_timer, '200')
-
```
Comment (landauermax) [inline aecid-testsuite/unit/analysis/NewMatchPathDetectorTest.py:251]: I am not sure if we should delete these tests. As stated in the description, these are tests for something that is not yet implemented, but they might be useful in the future. @dITySoftware could you please confirm this or should we delete the tests (and all the other commented out parts of the code in this PR?)

File: aecid-testsuite/unit/parsing/IpAddressDataModelElementTest.py [hunk 2]
```diff
@@ -250,20 +250,6 @@ def run():
...
-        # t = timeit.timeit(setup=_setup192, stmt="run()", number=times)
-        # print("time: ", t)
-        # print()
-        # print("0.0.0.0 is run 300.000 times.")
-        # t = timeit.timeit(setup=_setup0, stmt="run()", number=times)
-        # print("time: ", t)
-        # print()
-        # print("255.255.255.255 is run 300.000 times.")
-        # t = timeit.timeit(setup=_setup255, stmt="run()", number=times)
-        # print("time: ", t)
```
Comment (dITySoftware) [inline aecid-testsuite/unit/parsing/IpAddressDataModelElementTest.py:266]: Please do not delete the timeit tests. Those are valid, but commented out to speed up CI runs.

File: aecid-testsuite/unit/util/SecureOSFunctionsTest.py [hunk 3]
```diff
@@ -19,15 +19,6 @@ class SecureOSFunctionsTestLocal(TestBase):
...
     """This test case is commented out, because it is still not implemented."""
-    # '''
-    # A file is tried to be opened by using the secure function.
-    # '''
-    # def test1_secure_open_file(self):
-    #   error = sys.stderr = StringIO()
-    #   # if an exception is thrown, the test fails.
-    #   secure_open_file(b'/etc/aminer/conf-enabled/Readme.txt', os.O_RDONLY)
-    #   SecureOSFunctions.no_secure_open_warn_once_flag = True
-    #   self.assertEqual(error.getvalue(), '')
```
Comment (dITySoftware) [inline aecid-testsuite/unit/util/SecureOSFunctionsTest.py:30]: I am not sure about this one - @whotwagner will be working on that method at some point.

File: aecid-testsuite/unit/analysis/NewMatchPathDetectorTest.py (hunk 1 above)
Comment (dITySoftware) [reply aecid-testsuite/unit/analysis/NewMatchPathDetectorTest.py:251]: Everything aside the timeit parts and secure_open_file test can be deleted.

**Comment by landauermax** [comment]
Thanks for the quick response. @whotwagner please change the PR accordingly.

**Pull request closed without merging**